src/lib/openapi/index.ts

```diff
@@ -328,7 +344,7 @@ export const createOpenApiSchema = ({
     unleashUrl,
     baseUriPath,
 }: Pick<IServerOption, 'unleashUrl' | 'baseUriPath'>): Omit<
-    OpenAPIV3.Document,
+    OpenAPIV3DocumentWithServers,
```

Comment:
Servers are always returned from this function, so adding this fixes some null pointer compilation errors